src/Serialization/main.jl

```diff
@@ -239,6 +251,14 @@ function load_object(s::DeserializerState, T::Type, params::Any, key::Union{Symb
   end
 end
 
+function load_attrs(s::DeserializerState, obj::T) where T
+  load_node(s, :attrs) do _
```

Comment:
```suggestion
  haskey(s, :attrs) && load_node(s, :attrs) do _
```
this makes it possible to load old files that have no `:attrs` saved